lib/google_search_data_viewer_worker/keywords/helpers/job_creation.ex

```diff
@@ -0,0 +1,12 @@
+defmodule GoogleSearchDataViewerWorker.Keywords.JobCreationHelper do
```

Comment:
The name of this module does not match the file name. Please update it. I think the file name should be 
```
lib/google_search_data_viewer_worker/keywords/helpers/job_creation_helper.ex
```